File: GestaoHYS/GestaoHYS.Core/Models/PartyWithholdingTaxSchemas.cs
using Newtonsoft.Json;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace GestaoHYS.Core.Models
{
    public class PartyWithholdingTaxSchemas 
    {
        [Column("id")]
        [JsonProperty("id")]
        public string id { get; set; }

        [Column("partyWithholdingTaxSchemaKey")]
        [JsonProperty("partyWithholdingTaxSchemaKey")]
        public string partyWithholdingTaxSchemaKey { get; set; }

        [Column("description")]
        [JsonProperty("description")]
        public string description { get; set; }
    }
}

File: GestaoHYS/GestaoHYS.API/ClientApp/src/app/_models/PartyWithholdingTaxSchemas.ts
export class PartyWithholdingTaxSchemas{
    id: string;
    description: string;
    partyWithholdingTaxSchemaKey: string;
  
    constructor(id : string, name: string, key: string){
        this.id = id;
        this.description = name;
        this.partyWithholdingTaxSchemaKey = key;
    }
}